subsys/mpsl/Kconfig

```
@@ -6,8 +6,31 @@
 
 menu "Nordic MPSL"
 
+config MPSL
+	bool "Nordic Multi Protocol Service Layer (MPSL)"
+	select ZERO_LATENCY_IRQS
+	select IEEE802154_NRF5_EXT_IRQ_MGMT if IEEE802154_NRF5
+	select NRF_802154_MULTIPROTOCOL_SUPPORT if NRF_802154_RADIO_DRIVER
+	select MULTITHREADING_LOCK
+	depends on (SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET)
+	help
+	  Use Nordic Multi Protocol Service Layer (MPSL) implementation,
+	  providing services for single and multi-protocol implementations.
+
+if MPSL
+
+choice MPSL_BUILD_TYPE
+	prompt "MPSL build type"
+	default MPSL_BUILD_TYPE_LIB
+
+config MPSL_BUILD_TYPE_LIB
+	bool "Use library"
```

Just `library`.

In `menuconfig`, this looks cleaner `MPSL build type (library)` than `MPSL build type (Use library)`.

```suggestion
	bool "library"
```